### ISSUE TITLE
Normalization

### ISSUE THREAD

**Issue by Kong26** [yongqyu/gc-mc-pytorch#2]
구현 필요

**Comment by yongqyu** [comment]
```python    
def normalize(self, mx):
    """Row-normalize sparse matrix"""
    rowsum = torch.sum(mx, 0)
    r_inv = torch.pow(rowsum, -0.5)
    r_inv[torch.isinf(r_inv)] = 0.
    r_mat_inv = torch.diag(r_inv)
    colsum = torch.sum(mx, 1)
    c_inv = torch.pow(colsum, -0.5)
    c_inv[torch.isinf(c_inv)] = 0.
    c_mat_inv = torch.diag(c_inv)

    mx = torch.matmul(mx, r_mat_inv)
    mx = torch.matmul(c_mat_inv, mx)
    return mx
```
